roles/ceph-osd/tasks/scenarios/raw_multi_journal.yml

```diff
@@ -26,4 +27,13 @@
     - raw_multi_journal
     - not osd_auto_discovery
 
+- name: fail if ceph-disk cannot prepare an OSD
+  fail:
+    msg: "ceph-disk failed to prepare an OSD"
```

Comment:
Should we suggest an action here? A hint to look at the "prepare osd disk(s)" task output and fix and/or remove the malfunctioning device might be helpful.